source/Outfit.cpp

```diff
@@ -87,7 +99,18 @@ void Outfit::Load(const DataNode &node)
 				licenses.push_back(grand.Token(0));
 		}
 		else if(child.Size() >= 2)
+		{	
 			attributes[child.Token(0)] = child.Value(1);
+			// These outfits are not allowed to have values <= -1. Catch any
+			// violations upon loading the game so that the corrections don't
+			// need to be repeatedly calculated later.
+			if(PROTECTION_TYPES.find(child.Token(0)) != PROTECTION_TYPES.end())
+				if(child.Value(1) <= -1.)
+				{
+					child.PrintTrace("Skipping illegal negative attribute:");
```

Comment:
If this guard is before the attribute gets set, we avoid removing a base or plugin value if a later plugin sets an unsupported value.